v5/releases/kernel3.2.0.rst

```diff
@@ -0,0 +1,39 @@
+=========================
+PROS Kernel 3.2.0 Release
+=========================
+
+.. post:: 7 September, 2019
+   :tags: blog, kernel-release
+
+New Features:
+
+- Upgrade LVGL to version 5.3
+- Add an alternate :code:`pros::Task` constructor that takes only a task function, parameters, and a name.
+- Add an alternate API for generic serial communications over the smart ports
+- Add a better data abort handler that prints a stack trace
+
+Usability Improvements:
+
+- Add Makefile variable for excluding libraries from the cold image
+- Improve Makefile compile speed
+- Support GCC 8.3 out of the box
+- Add a default .gitignore to new projects
+- Changes in header files will now be recompiled without having to clean
+- Replace separate :code:`initialize.cpp`, :code:`autonomous.cpp`, and :code:`opcontrol.cpp` files with a single :code:`main.cpp` file for new projects
+
+Bugfixes:
+
+- Make :code:`vision_read_by_sig` properly transform coordinates and respect array boundaries
+- Make VDML/ADI :code:`errno` values more consistent
+- Other assorted VDML/ADI bugfixes
+- Fix generic serial communications driver
+- Fix C++ file I/O
+
+Important upgrade notes
```

Comment:
This part might be better off at the top of the release notes, the changelog details likely won't be super important to most people so they might ignore the whole of the release notes based on that being at the beginning.

Reply:
Fair enough. I was mostly copying the format we've used in previous release posts